src/components/PeopleLink/PeopleLink.tsx

```diff
@@ -0,0 +1,48 @@
+import cn from 'classnames';
+import { Link, useParams } from 'react-router-dom';
+import { Person } from '../../types';
+
+type PeopleLinkProps = {
+  person: Person;
+};
+
+export const PeopleLink: React.FC<PeopleLinkProps> = ({ person }) => {
+  const { name, sex, born, died, motherName, fatherName, mother, father } =
+    person;
+  const { slug } = useParams();
+  const female = sex === 'f';
```

Comment:
Better to use `isFemale` name - to indicate that it's boolean. Usage of `female` name feels like it might be some string/object/etc.